docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -132,3 +178,19 @@ $ open -a "Teleport Connect" --args --insecure
 # Passing the flag to the executable directly:
 $ /Applications/Teleport\ Connect.app/Contents/MacOS/Teleport\ Connect --insecure
 ```
+</TabItem>
+<TabItem label="Linux">
+From a terminal, open Teleport Connect with the `--insecure` flag:
+
+```code
+$ teleport-connect --insecure
+````
+</TabItem>
+<TabItem label="Windows">
+From the Command prompt, open Teleport Connect with the `--insecure` flag:
```

Comment:
By the Command prompt you mean the Windows program, right? I just booted up the VM to check the exact name of it.

```suggestion
From the Command Prompt, open Teleport Connect with the `--insecure` flag:
```